python/res/enkf/local_ministep.py

```diff
@@ -58,6 +59,10 @@ def getName(self):
         """ @rtype: str """
         return self.name()
 
+    def getObsData(self):
```

Comment:
use snake_eyes?

Reply:
Agreed, but do we have any "official" guidelines for what we do when adding new features to files with wrong formatting?